draft-ietf-quic-recovery.md

```diff
@@ -222,13 +222,17 @@ the peer spurious timeouts.  The maximum ack delay is communicated in the
 `max_ack_delay` transport parameter and the default value is 25ms.
 
 An acknowledgement SHOULD be sent immediately upon receipt of a second
-packet but the delay SHOULD NOT exceed the maximum ack delay. QUIC recovery
-algorithms do not assume the peer generates an acknowledgement immediately when
-receiving a second full-packet.
+ack-eliciting packet. QUIC recovery algorithms do not assume the peer sends an
+an ACK immediately when receiving a second ack-eliciting packet.
 
 Out-of-order packets SHOULD be acknowledged more quickly, in order to accelerate
 loss recovery.  The receiver SHOULD send an immediate ACK when it receives a new
-packet which is not one greater than the largest received packet number.
+packet which is not the next expected one. That is, its packet number is not one
+greater than the largest received packet number.  A receiver MAY immediately
+acknowledge subsequent ack-eliciting packets after first receiving a packet out
+of order. A receiver SHOULD NOT send an immediate ACK any time there is a gap
+in the ACK frame being sent, because that will cause it to send an ACK in
+response to every packet for at least an RTT after a single packet loss.
```

Comment:
I don't understand what the text here is trying to say.  I'll suggest rewording for text "A receiver MAY ... single packet loss", see if this matches what you're getting at.

"A receiver MAY send acknowledgements immediately for the next few packets that are received, but the receiver SHOULD revert back to delaying acknowledgements after sending a few immediate acknowledgements."